gef.py

```diff
@@ -6289,20 +6304,25 @@ def __init__(self):
     @only_if_gdb_running
     def do_invoke(self, argv):
         location = None
+        show_opcodes = False
 
         kwargs = {}
         for arg in argv:
             if "=" in arg:
                 key, value = arg.split("=", 1)
                 kwargs[key] = value
 
+            elif "opcodes".startswith(arg.lower()):
+                show_opcodes = True
+
             elif location is None:
                 location = parse_address(arg)
 
         location = location or current_arch.pc
         length = int(kwargs.get("length", get_gef_setting("context.nb_lines_code")))
 
         for insn in capstone_disassemble(location, length, skip=length*self.repeat_count, **kwargs):
+            insn.set_opcodes_visibility(show_opcodes)
             text_insn = str(insn)
```

Comment:
Here: We could do something like `"{:o}".format(insn)`. The `__format__` method can be written to read these options

Reply:
Thanks, this is better. I've updated it along with @hugsy's suggestion, so that we can do either `{:o}` to print with all opcodes, or `{:4o}` to show for example 4 bytes of the opcodes.